src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
Not sure that this console.error is necessary, the error is thrown very shortly afterwards. And if it is useful, it has just occurred to me that it should probably log error.stack if available.